### ISSUE TITLE
Too many connections - Mobile 

### ISSUE THREAD

**Issue by Djidjisven** [mchristopher/PokemonGo-DesktopMap#727]
Hello,

When I use the map on my phone, after a few secondes, the map stop working. If I reload, I have this message : "Too many connections ! The tunnel session '.....' has violated the rate-limit of 20 connections per minute. (...)"

Someone know how I can solve this issue ?

Thanks
Dj

**Comment by abuch47** [comment]
Im using chrome mobile and had the same thing. Worked fine for 12 hours then all of a sudden got this message as well.

**Comment by abuch47** [comment]
I have 50 going at home but only one mobile uplink

**Comment by Djidjisven** [comment]
I'm using multi accounts to scan several areas. So when I put it on my phone, I can see these multi areas but after a few secondes this error appears. That's maybe the reason?
 I have to use only one account if i want to put it on my phone without this "Too many connections" error ?

**Comment by pill0ws** [comment]
lol 50 connections? tell me you don't mean that you are running 50 instances of the App. I may rest easier knowing there aren't people out there blatantly hammering their servers as hard as their home PC can manage it

**Comment by Djidjisven** [comment]
I was running 4 instances of the app at home with one mobile link when I got this error. So I thought the "Too many  connections" error on my phone was caused by that. So I tried with only one instance but still have the same problem... I dont know what I have to do.

**Comment by JMHendon** [comment]
I have the same issue today - I'm running one instance of the app at home and am using the mobile version on Safari.

**Comment by thaaatsjoe** [comment]
Using safari or chrome on 6s+ from my win10 rig gives me this problem. But so far not from my win7 rig, having a different issue there (works at first, then I get "waiting"). Works on both computers for a few minutes before needing to log out and back in. Just one instance. Separate account. Version 0.2.1. 

I guess it should be noted that on the win10 rig I've installed jsnodes and python 2.7 for some other PoGoMap examples.
And on the win7 rig I did nothing but run the exe.
Not sure if any of that is usful.
